src/VisualStudio/Core/Def/Implementation/ProjectSystem/CPS/ICodeModelFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.LanguageServices.ProjectSystem
```

Comment:
I have kept the new interfaces in the CPS folder, but they are not CPS specific at all (any project system can use them). I'll move them to the root ProjectSystem folder.